demo/README.md

```diff
@@ -1,32 +1,97 @@
-## Goal: Bring up Clock Auction Demo Application
+# Clock Auction Demo Application
 
-###  Prerequisites
+##  Prerequisites
 - It is assumed that Fabric Private Chaincode repository on your machine is installed in $FPC_PATH.
 - JSON processor, jq is installed.
+- FPC Peer Docker Image has been created as per the FPC Network
+[Instructions](../utils/docker-compose/README.md#Steps) in Step 1.
+- Relevant Fabric binaires and docker images have been downloaded as per FPC
+Network [Instructions](../utils/docker-compose/README.md#Steps) in Step 2.
 
-Instructions to bring up end-to-end application to be added here.
+The Auction Demo has multiple components, a UI, a backend server that proxies calls
+to an FPC network, a FPC network and a chaincode to execute the auction logic.
 
-## Backend client
+## Bring Up the Demo End To End
+### Setup
+To set up all components at once run the start script.
+```
+scripts/startFPCAuctionNetwork.sh
+```
+**NOTE** The above [script](scripts/startFPCAuctionNetwork.sh) will bring up a
+fresh FPC Network and generate new credentials using the FPC Network Setup
+[scripts](../utils/docker-compose/scripts). Therefore this script should only be
+run when no other fabric network is running to avoid port collisions.
+
+Both the frontend and fabric-gateway [expose ports](docker-compose.yml) and are
+accessible on the host machine. The frontend can be accessed at `localhost:5000`
+and the fabric-gateway can be accessed at `localhost:3000`.
+
+### Teardown
+To bring down all of the components and the underlying FPC network run the
+following script.
+```
+scripts/teardown.sh
+```
+**NOTE** The script will run the [teardown script](../utils/docker-compose/scripts/teardown.sh)
+in the FPC Network scripts with the `--clean-slate` flag which will delete all
+the volumes associated with the network.
+
+## Manually Bring Up The Components
+
+### 1. FPC Network
+This demo requires the use of a FPC Network. Follow the [instructions](../utils/docker-compose/README.md#Steps)
+in this repo to bring one up. The demo directory is [mounted](../utils/docker-compose/network-config/docker-compose.yml)
+into the peer to make installing chaincode easy.
+
+### 2. Install the Auction Chaincode
+Install the mock golang chaincode for the demo.
+
+To install the chaincode you can run [installCC script] (scripts/installCC.sh)
+```
+cd $FPC_PATH/demo
+scripts/installCC.sh
+```
+If you prefer to install it manually use the following steps.
+
+1. Exec into the peer container
+```
+docker exec -it peer0.org1.example.com bash
+```
+
+2. Install the mockcc
+```
+${PEER_CMD} chaincode install -n mockcc -v 1.0 --path github.com/hyperledger-labs/fabric-private-chaincode/demo/chaincode/golang/cmd -l golang
```

Comment:
Not sure where the variable, PEER_CMD is set.  It may be helpful to call it out here or set it here.

Reply:
I can add a little to the documentation saying that the environment variables, if not explicitly set here are set in the FPC network setup. It is explained in the FPC network docs, so I don't want to replicate all the information because I think it will be harder to maintain in the future.

Reply:
I added a link to the FPC documentation and mention there are environment variables already set in the container to make the steps more convenient in a fixup commit